greedybear/settings.py

```diff
@@ -56,6 +62,8 @@
     "gui.apps.GuiConfig",
     "api.apps.ApiConfig",
     "greedybear.apps.GreedyBearConfig",
+    # db backups
+    "dbbackup",
```

Comment:
can you explain advantages of this external app when we have https://docs.djangoproject.com/en/4.0/ref/django-admin/#dumpdata as default in Django?

Reply:
Oh, I haven't looked into that yet. That seems like a better alternative than adding another dependency. I'll try implementing it with `dumpdata` instead 👍🏼

Reply:
Hello Sir, I've updated the implementation with `dumpdata`. Please review changes :)